.github/workflows/bump-version.yml

```diff
@@ -1,44 +1,106 @@
+name: Bump Version
+
 on:
-    pull_request:
-      branches:
-        - main
-      types: [closed]
-  
+  pull_request:
+    types: [closed]
```

Comment:
I noticed a bug here.

1. **Trigger:** The workflow is initiated when a pull request is closed on the `main` branch.

This means that any closed PR would run this logic, potentially.

Reply:
Good catch. I think I had the 
```    if: github.event_name == 'pull_request' && github.event.action == 'closed'
```
line before and it seems to have been nixed somewhere in getting the 'happy flow' to work